components/caseStudyDetails/ProcessIcons.tsx

```diff
@@ -0,0 +1,15 @@
+const ProcessIcons = ({ Process }: { Process: React.FC }) => {
```

Comment:
You don't need to capitalize "Process."

Reply:
The process here is a React component so without capitalizing it causes an error.